public/en/platform/protect-sdk-php/configuration-documentation.md

```diff
@@ -84,3 +84,5 @@ use NS8\ProtectSDK\Config\Manager as ConfigManager;
 // Does Foo config value exist?
 $doesFooExist = ConfigManager::doesValueExist('foo');
 ```
+
+
```

Comment:
This doesn't seem super necessary but I guess there's no harm in a little extra white space.